### DISCUSSION TITLE
Adding the function of input source switch (like from HDMI to usbc)

### DISCUSSION THREAD

**Discussion by pkuanjie** [MonitorControl/MonitorControl#871] [Ideas]
It would be awesome and very useful!

**Comment by waydabber** [comment]
Hey @pkuanjie - this is on the feature request list, hopefully we'll get there, maybe in the next push towards 5.0.0.

https://github.com/MonitorControl/MonitorControl/issues/720

**Reply by smachiz** [reply]
would love as well!  I found https://github.com/kfix/ddcctl that may work for you too @pkuanjie if you have a dell display - but command line only.  DDM for mac is poorly written and includes a lot I don't want - the only part I really want is to be able to switch inputs with configurable hot keys!

But command line is helpful.  Maybe the code is helpful to look at for importing into MOnitorControl :)

**Reply by pkuanjie** [reply]
Currently, I am using https://github.com/waydabber/m1ddc in the command line and keyboard maestro to set a hotkey to enable the switch. I am always living in the command line so this solution works perfectly for me.
